src/components/Todo/todo.scss

```diff
@@ -25,6 +25,39 @@
 .todo-buttons {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  align-items: flex-start;
   justify-content: center;
+}
+
+.todo-select {
+  width: 150px;
+  max-width: 200px;
+  padding: 7px 20px;
+  text-align: center;
+  -moz-appearance: none;
+	-webkit-appearance: none;
+  border: 1px solid black;
+  border-radius: 5px;
```

Comment:
I don't see the rounded borders on the mockup

Reply:
deleted